tree/ntupleutil/v7/test/ntuple_inspector.cxx

```diff
@@ -813,3 +813,23 @@ TEST(RNTupleInspector, MultiColumnRepresentations)
    EXPECT_EQ(ENTupleColumnType::kReal16, px1Inspector.GetType());
    EXPECT_EQ(1u, px1Inspector.GetNElements());
 }
+
+TEST(RNTupleInspector, FieldsTreeAsDot)
```

Comment:
I believe the output from `PrintFieldsTreeAsDot` for this test is not all too large, right? In that case, perhaps it would be nice to (also) have a 1-to-1 string comparison instead of only checking the substrings below.